src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -95,7 +95,7 @@ public void Abort()
 
       private void logLine(string textToLog)
       {
-         logText($"{Environment.NewLine}{textToLog}");
+         logText($"{Environment.NewLine}{textToLog}${Environment.NewLine}");
```

Comment:
We definitely need the trailing line because if we write a logLine, then the logger writes next it will be appended to the same line. The logger assumes that a newline was at the end of the previous line.